install_sct.bat

```diff
@@ -42,7 +42,7 @@ echo ### Installation finished!
 echo:
 echo To use SCT's command-line scripts in Command Prompt, please follow these instructions:
 echo:
-echo 1. Open the Start Menu -^> Type 'path' -^> Open 'Edit the System Environment Variables'
+echo 1. Open the Start Menu -^> Type 'path' -^> Open 'Edit environment variables for your account'
 echo 2. Click the 'Environment variables...' button
 echo 3. Under the section 'User variables for ____', highlight the 'Path' entry, then click the 'Edit...' button.
 echo 4. Click 'New', then copy and paste this directory:

```

Comment:
Oops... Just realized that the interface for the "Edit environment variables for your account" page has a different UI than the "Edit the System Environment Variables" page, so step 2 is no longer needed.

```suggestion
echo 1. Open the Start Menu -^> Type 'path' -^> Open 'Edit environment variables for your account'
echo 2. Under the section 'User variables for ____', highlight the 'Path' entry, then click the 'Edit...' button.
echo 3. Click 'New', then copy and paste this directory:
```